oap-server/server-storage-plugin/storage-iotdb-plugin/pom.xml

```diff
@@ -49,6 +49,16 @@
                     <groupId>net.jpountz.lz4</groupId>
                     <artifactId>lz4</artifactId>
                 </exclusion>
+                <!-- logback has runtime negative impact about performance, -->
```

Comment:
IMHO this is a kind of arbitrary decision and we didn't even have a solid conclusion about the reason of https://github.com/apache/skywalking/pull/8194 , let's remove this kind of sentence, to me the only reason to exclude this is that we already have slf4j implementation (log4j2) in the classpath

Reply:
I added this, not @mrproliu 

I am thinking the performance impact is real, but we don't know why.

Reply:
> I added this, not @mrproliu

Don't take it personal, I was just commenting in the perspective of logback's community, no matter who, without benchmark test result / evidence, saying others have pool performance is not cool, I still remember someone said SkyWalking is pool performant in Twitter, but they were not aware of that they misused SkyWalking, I felt the same uncomfortable that time.

> I am thinking the performance impact is real, but we don't know why.

The performance impact is real, but we can only say it exists when logback + log4j2 + slf4j are all in the classpath, if we really want to put this sentence, we at least should test another case where we remove log4j2 and keep logback + slf4j.

Reply:
I can understand the statement to co-existing to make more accurate. And ref the PR with screenshots as prove.